draft-ietf-tls-esni.md

```diff
@@ -382,6 +382,11 @@ retry_configs
 decreasing order of preference, to be used by the client in subsequent
 connection attempts. These are known as the server's "retry configurations".
 
+If client offers the "encrypted_client_hello" extension and the client-facing
+server accepts, then the backend server MAY include an "encrypted_client_hello"
```

Comment:
This seems like it has to be a MUST based on the text below in which I assume rejection if it's absent.

Reply:
You're right, although the MUST is also conditioned on the backend server sending HRR.  Updated.